fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/SnsAmbassador.java

```diff
@@ -94,7 +94,7 @@ public void initialize(final long startTime, final long endTime) throws Internal
         }
 
         try {
-            CSns configuration = new ObjectInstantiation<>(CSns.class).readFile(ambassadorParameter.configuration);
+            CSns configuration = new ObjectInstantiation<>(CSns.class, log).readFile(ambassadorParameter.configuration);
```

Comment:
Good idea, I'll have a check if/how much log files are polluted with this change.